xml/dm_registers.xml

```diff
@@ -586,19 +611,19 @@
         When the system bus master is busy,
         writes to this register will set \Fsberror.
 
-        If \Fsberror is 0 and \Fsbautoread is set then the system bus
+        If \Fsberror is 0 and \Fsbreadonaddr is set then the system bus
```

Comment:
Should this say "... is set then a *write* to this register will cause the system bus master to start a read at the newly written address.

`\Faddress` is the wrong macro. It should be `\Fsbaddresszero`.

Reply:
Yes. Fixing.